libexec/alluxio-config.sh

```diff
@@ -41,6 +41,11 @@ fi
 
 . "${ALLUXIO_CONF_DIR}/alluxio-env.sh"
 
+if [[ -n "${ALLUXIO_MASTER_ADDRESS}" ]]; then
+  echo "ALLUXIO_MASTER_ADDRESS is deprecated, please switch to ALLUXIO_MASTER_HOSTNAME."
```

Comment:
echo "ALLUXIO_MASTER_ADDRESS is deprecated since version 1.1 and will be remove in version 2.0."
echo "Please use ALLUXIO_MASTER_HOSTNAME" instead."